written_communications_functions.php

```diff
@@ -207,9 +207,9 @@ function getProfileFieldValue($fieldNameIntern, $userId)
 }
 
 // Check if own templates are available and set template path
-if(is_dir('../../adm_my_files/download/MSWord_Templates'))
+if(is_dir('../../adm_my_files/download_clb/MSWord_Templates'))
```

Comment:
**download_clb** is your specific download folder. Use `TableFolder::getRootFolderName()` to get the organization specific download folder.

e.g.
`'../../adm_my_files/'. TableFolder::getRootFolderName() .'/MSWord_Templates'`